CMakeLists.txt

```diff
@@ -133,23 +133,23 @@ option(INCLUDE_LINUX_I2C "Include I2C support" YES)
 IF (INCLUDE_LINUX_I2C)
 	CHECK_INCLUDE_FILES ("sys/types.h;linux/i2c-dev.h;linux/i2c.h" HAVE_LINUX_I2C_H)
 	IF (HAVE_LINUX_I2C_H)
-		message(STATUS "Building with I2C support")
+		message(SEND_ERROR "Building with I2C support")
 		add_definitions(-DHAVE_LINUX_I2C)
 	ELSE()
-		message(WARNING "I2C support disabled: headers not found!")
+		message(SEND_ERROR "I2C support disabled: headers not found!")
```

Comment:
This should be a warning as well
Default people do not disable this option, and if you build on a normal system, it will fail because there is no SPI/I2C